packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -944,12 +1001,23 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
     return tableWidth;
   }
 
+  Widget _buildItemPinned(BuildContext context, int index) {
```

Comment:
instead of creating a new method, just add an optional named parameter `bool pinned = false` to `_buildItem`